holoviews/plotting/bokeh/plot.py

```diff
@@ -74,6 +74,11 @@ def document(self):
         return self._document
 
 
+    @property
+    def id(self):
+        return self.state._id
```

Comment:
Sure using ``_id`` is fine here? I assume this is ``_id`` in bokeh not holoviews...

Reply:
Yeah, this is bokeh, not sure why bokeh doesn't make ``_id`` public, it's pretty central to a lot of things. I'll double check there isn't a public API.

Reply:
``obj.ref['id']`` works